### PR TITLE
Fe application

### PR DESCRIPTION
**Pull request by Christonn93**

### CODE REVIEW

**Review by Tactikerl — APPROVED** [review]
Good work on this everyone. However, I have some issues with the green colour used in the Client Application view, will confer with SM and design team on this.

File: pages/client/application/index.html [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+              <p class="mb-2" id="applicationText">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed euismod, nisi id tempus porttitor, risus erat euismod risus, at dapibus neque turpis vel est. Sed accumsan, nibh id euismod bibendum, elit est placerat nibh, euismod rhoncus nisl nisl in est.</p>
+              <br />
+              <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed euismod, nisi id tempus porttitor, risus erat euismod risus, at dapibus neque turpis vel est.</p>
+            </div>
+
+            <h5 class="font-bold my-4" id="apllicantsName">Applicants/Student Name</h5>
+
+            <div class="mb-4">
+              <div>
+                <img src="https://via.placeholder.com/20" alt="Placeholder Icon" />
```
Comment (Christonn93) [inline pages/client/application/index.html:52]: Links need a class to remove the green color on the links.
Try adding `class="nav-link text-black"`

File: pages/client/application/index.html [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+              <div>
+                <img src="https://via.placeholder.com/20" alt="Placeholder Icon" />
+                <a href="#" id="applicationLink2">Filename (link added by student)</a>
+              </div>
+            </div>
+
+            <form id="clientApplication">
+              <a href="#" class="btn btn-primary px-5 m-1" id="clientApplicationOffer">Offer</a>
+              <a href="#" class="btn btn-secondary px-5 m-1" id="clientAplicationRejects">Rejects</a>
```
Comment (Christonn93) [inline pages/client/application/index.html:64]: the clas on the button is wrong. It need to have `btn-theme-secondary`

File: pages/client/application/index.html [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+              <br />
+              <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed euismod, nisi id tempus porttitor, risus erat euismod risus, at dapibus neque turpis vel est.</p>
+            </div>
+
+            <h5 class="font-bold my-4" id="apllicantsName">Applicants/Student Name</h5>
+
+            <div class="mb-4">
+              <div>
+                <img src="https://via.placeholder.com/20" alt="Placeholder Icon" />
+                <a href="#" id="applicationLink1">Link (link added by student)</a>
```
Comment (Christonn93) [inline pages/client/application/index.html:53]: Links need a class to remove the green color on the links.
Try adding `class="nav-link text-black"`

File: pages/client/application/index.html [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+              </div>
+
+              <div>
+                <img src="https://via.placeholder.com/20" alt="Placeholder Icon" />
+                <a href="#" id="applicationLink2">Filename (link added by student)</a>
```
Comment (Christonn93) [inline pages/client/application/index.html:58]: Links need a class to remove the green color on the links.
Try adding `class="nav-link text-black"`

File: pages/client/application/index.html [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+              </div>
+            </div>
+
+            <form id="clientApplication">
+              <a href="#" class="btn btn-primary px-5 m-1" id="clientApplicationOffer">Offer</a>
```
Comment (Tactikerl) [inline pages/client/application/index.html:63]: Wrong bootstrap class used here. Should be `btn-theme-Secondary` not `btn-primary`

File: src/js/api/application/fetchSingleApplication.js [hunk 6]
```diff
@@ -0,0 +1,14 @@
+import { displaySingleApplication } from '../../ui/application/singleApplication';
```
Comment (Christonn93) [inline src/js/api/application/fetchSingleApplication.js:1]: Missing jsdoc

File: pages/client/application/index.html (hunk 5 above)
Comment (Tactikerl) [reply pages/client/application/index.html:63]: @Christonn93

File: src/scss/custom/_index.scss [hunk 7]
```diff
@@ -0,0 +1,9 @@
+@import '../../../node_modules/bootstrap/scss/functions';
+
+.clientAplication {
```
Comment (Christonn93) [inline src/scss/custom/_index.scss:3]: these classes should be in a separate file that you import into index from the custom folder.

File: pages/client/application/index.html (hunk 5 above)
Comment (Tactikerl) [reply pages/client/application/index.html:63]: @asimenstad

**Review by Tactikerl — CHANGES_REQUESTED** [review]
@asimenstad The bootstrap class on line 63 in `pages/client/application/index.html` should be `btn-theme-Secondary` could you update this?

**Review by Tactikerl — APPROVED** [review]
Looks good

File: src/js/api/application/fetchSingleApplication.js (hunk 6 above)
Comment (Christonn93) [reply src/js/api/application/fetchSingleApplication.js:1]: I will add this when its merged.

File: src/scss/custom/_index.scss (hunk 7 above)
Comment (Christonn93) [reply src/scss/custom/_index.scss:3]: I will fix this when its merged

**Review by Pjatte1337 — APPROVED** [review]
Looks good

**Pull request merged**